backend/internal/github/userclient/userclient.go

```diff
@@ -149,3 +149,26 @@ func (api *UserAPI) ForkRepository(ctx context.Context, org, owner, repo, destNa
 
 	return nil
 }
+
+func (api *UserAPI) CreateFeedbackPR(ctx context.Context, owner, repo string) error {
+	endpoint := fmt.Sprintf("/repos/%s/%s/pulls", owner, repo)
+
+	//Initialize post request
+	req, err := api.Client.NewRequest("POST", endpoint, map[string]interface{}{
+		"title": "Feedback",
```

Comment:
hard coding "business logic" into our API layer is bad design. The parameters that you are using here in the request should be passed into the api function